package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nifti-reader-js",
-  "version": "0.6.2",
+  "version": "0.7.0",
```

Comment:
Recommend a version bump for this change. Since version is still < `1.0.0`, conventionally, this release could introduce breaking changes.